components/atoms/ChanceOfSuccess/ChanceOfSuccess.story.jsx

```diff
@@ -80,3 +80,13 @@ stories.add(
   ),
   {percy: {skip: true}},
 );
+
+stories.add(
+  'missing percent',
+  () => (
+    <ChanceOfSuccess
+      {...defaultProps({draft: false, running: false, compact: false})}
+    />
```

Comment:
Can we add a background color to this story and the others, similar to how we do in the markdown file for this component? It took me a bit to realize the text was there with it being white.

```suggestion
    <div style={{backgroundColor: 'black'}}>
      <ChanceOfSuccess
        {...defaultProps({draft: false, running: false, compact: false})}
      />
   </div>
```

Reply:
There's a background toggle at the top of the story view. If you add a fixed background it will eliminate the ability to test it on other background colors. I think it makes sense in the docs file as such a thing doesn't exist there.

Reply:
Ah I was not aware of this configuration. I just did some research and it looks like we add the `backgrounds` param in `config.js` and set the default to white. Would it be okay to do something like this just for this set of stories (and any others like it in the future):

```javascript
...
import {colors} from '~constants/js/colors';

const stories = storiesOf('Atoms/ChanceOfSuccess', module);

stories.addParameters({
  docs: {
    page: ChanceOfSuccessReadme,
  },
  backgrounds: [
    {name: 'White', value: colors['white']},
    {name: 'Parchment', value: colors['parchment']},
    // Or any other color that's not white
    {name: 'Royal', value: colors['royal'], default: true},
    {name: 'Midnight', value: colors['midnight']},
    {name: 'Middle Purple', value: colors['midnight-purple']},
  ]
});
...
```

Reply:
Also, looking at these colors configs I actually think there's a bug - I think `midnight-purple` is meant to be `middle-purple` as there's no `midnight-purple` in the `colors.js` constants.

Reply:
That approach makes the most sense to me. Perhaps we could make the array of objects shared.